src/gantt.scss

```diff
@@ -157,6 +167,18 @@ $handle-color: #ddd !default;
 			color: #dfe2e5;
 		}
 
+		.action{
```

Comment:
Can we make the action button a link instead of a big button? Also, the action must be configured from options, so that users can customise which button they want to use to trigger this action.